front/app/containers/Admin/projects/project/analysis/Insights/index.tsx

```diff
@@ -1,22 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import { Box, Text } from '@citizenlab/cl2-component-library';
 
-import useAnalysisSummaries from 'api/analysis_summaries/useAnalysisSummaries';
+import useAnalysisInsights from 'api/analysis_insights/useAnalysisInsights';
 
 import { useParams } from 'react-router-dom';
 
 import Summary from './Summary';
+import SummarizeButton from './SummarizeButton';
+import QuestionButton from './QuestionButton';
+import QuestionInput from './QuestionInput';
+import Question from './Question';
 
 const Insights = () => {
+  const [isQuestionInputOpen, setIsQuestionInputOpen] = useState(false);
   const { analysisId } = useParams() as { analysisId: string };
-  const { data: summaries, isLoading } = useAnalysisSummaries({
+  const { data: insights, isLoading } = useAnalysisInsights({
     analysisId,
   });
 
   return (
     <Box>
-      {!isLoading && summaries?.data?.length === 0 && (
+      <Box display="flex" gap="4px">
+        <SummarizeButton />
+        <QuestionButton onClick={() => setIsQuestionInputOpen(true)} />
+      </Box>
```

Comment:
These still deserve some styling love, but I assume you're thinking the same :)

Reply:
Indeed, I agree. I'll think about it some more and ask Rob for input if I don't come up with a good approach. If you have a good idea, lmk :)